origocli/commands/pipelines/instances.py

```diff
@@ -0,0 +1,195 @@
+        arn = self.opt("pipeline-arn")
+        instances, error = self.sdk.get_pipeline(arn).list_instances()
+        if error:
+            self.log.exception(error)
+            return
+        data = list(map(lambda x: x.__dict__, instances))
+        out.add_rows(data)
+        self.print(f"Instances with arn: {arn}", out)
+
+
+class PipelineInstanceLs(BasePipelinesCommand):
+    """usage:
+      origo pipelines instances ls [(<dataset-id> <version>)] [options]
+
+    options:
+      -d --debug
+      --format=<format>
+    """
+
+    def default(self):
+        out = create_output(self.opt("format"), "pipelines_instances_config.json")
+        dataset_id = self.arg("dataset-id")
+        version = self.arg("version")
+        data = self.sdk.list(PipelineInstance)
+        title = ""
+        if dataset_id and version:
+            data = [
+                instance
+                for instance in data
+                if instance["datasetUri"] == f"output/{dataset_id}/{version}"
+            ]
+            out.add_rows(data)
+            title = "Pipeline instance"
+        else:
+            out.add_rows(data)
+            title = "Pipeline instances"
+        self.print(title, out)
+
+
+class PipelineInstancesCreate(BasePipelinesCommand):
+    """
+    usage:
+      origo pipelines instances create - [options]
+      origo pipelines instances create <file> [options]
+
+    options:
+      -d --debug
+      --format=<format>
+    """
+
+    def default(self):
+        out = create_output(self.opt("format"), "pipelines_instances_config.json")
+        self.log.info("PipelineInstancesCreate")
+        content = self.handle_input()
+        resource, error = PipelineInstance.from_json(self.sdk, content).create()
+        if error:
+            self.print(f"ERROR: {error} from: {content}")
+            self.log.exception(error)
+            return
+        originaldata = json.loads(content)
+        data = {
+            "id": json.loads(resource),
+            "datasetUri": originaldata["datasetUri"],
+            "pipelineArn": originaldata["pipelineArn"],
+        }
+        out.output_singular_object = True
+        out.add_row(data)
+        self.print("Created resources", out)
+
+
+class PipelineInstances(BasePipelinesCommand):
+    """
+    usage:
+      origo pipelines instances ls [(<dataset-id> <version>)] [options]
+      origo pipelines instances wizard
+      origo pipelines instances create - [options]
+      origo pipelines instances create <file> [options]
+
+    options:
+      -d --debug
+      --format=<format>
+    """
+
+    def __init__(self, sdk):
+        super().__init__(sdk)
+        self.sub_commands = [
+            PipelineInstancesCreate,
+            PipelineInstanceLs,
+            PipelineInstanceWizard,
+        ]
+
+    def default(self):
+        self.help()
+
+
+class PipelineInstanceWizard(BasePipelinesCommand):
+    """usage:
```

Comment:
This is using the old pipeline config? Should we leave this out for now, or at least remove it from the sub commands above?

Reply:
Added a ticket to ensure the command is updated